src/main/java/com/ibm/cloud/sdk/core/http/HttpClientSingleton.java

```diff
@@ -352,7 +351,6 @@ private OkHttpClient reconfigureClientInterceptors(OkHttpClient client, String i
       for (Iterator<Interceptor> iter = builder.interceptors().iterator(); iter.hasNext(); ) {
         Interceptor element = iter.next();
         if (interceptorToRemove.equals(element.getClass().getSimpleName())) {
-          LOG.log(Level.FINE, "Removing interceptor " + element.getClass().getName() + " from http client instance.");
```

Comment:
I'm curious - why remove these logs?

Reply:
I think I just didn't find those all that useful.  That's my story and I'm sticking to it.   I can easily add them back if needed.

Reply:
Sounds good haha, just wanted to understand the justification. I think that's sufficient 🙂